documentation/puppet_server_metrics_performance.markdown

```diff
@@ -73,6 +77,6 @@ Puppet Server also requests facts as HTTP requests while handling a node request
 
 ### Memory leaks and usage
 
-A memory leak or increased memory pressure can stress Puppet Server's available resources. In this case, the Java VM will spend more time doing garbage collection, causing the GC time and GC CPU % metrics to increase. These metrics are available in the [developer dashboard][] and [status API][] endpoint, as well as in the `/metrics/v1/mbeans` endpoint.
+A memory leak or increased memory pressure can stress Puppet Server's available resources. In this case, the Java VM will spend more time doing garbage collection, causing the GC time and GC CPU % metrics to increase. These metrics are available in the [developer dashboard][] and [status API][] endpoint, as well as in the mbeans metrics available from both the [`/metrics/v1/mbeans`](./metrics-api/v1/metrics_api.markdown) or [`/metrics/v2/mbeans`](./metrics-api/v2/metrics_api.markdown) endpoints.
```

Comment:
The path portion of the endpoints change from `v1` to `v2` so that its `metrics/v1/mbeans` but `metrics/v2/read` (most likely).